src/libopensc/types.h

```diff
@@ -52,9 +52,11 @@ typedef unsigned char u8;
 
 /* When changing this value, pay attention to the initialization of the ASN1
  * static variables that use this macro, like, for example,
- * 'c_asn1_supported_algorithms' in src/libopensc/pkcs15.c
+ * 'c_asn1_supported_algorithms' in src/libopensc/pkcs15.c,
+ * src/libopensc/pkcs15-prkey.c and src/libopensc/pkcs15-skey.c
+ * `grep "src/libopensc/types.h SC_MAX_SUPPORTED_ALGORITHMS  defined as"'
  */
-#define SC_MAX_SUPPORTED_ALGORITHMS	8
+#define SC_MAX_SUPPORTED_ALGORITHMS	16
```

Comment:
clang-format suggested the following change:


```
-#define SC_MAX_SUPPORTED_ALGORITHMS	16
+#define SC_MAX_SUPPORTED_ALGORITHMS 16
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/types.h -->